lib/modules/platform/gitea/index.ts

```diff
@@ -154,6 +155,24 @@ async function lookupLabelByName(name: string): Promise<number | null> {
   return labelList.find((l) => l.name === name)?.id ?? null;
 }
 
+async function fetchRepositories(topic?: string): Promise<string[]> {
+  const repos = await helper.searchRepos({
+    uid: botUserID,
+    archived: false,
+    ...(topic && {
+      topic: true,
+      q: topic,
```

Comment:
can we only search one topic at a time?

Reply:
yes, Gitea API supports searching only by one topic at a time